services/surfaceflinger/SurfaceFlinger.cpp

```diff
@@ -1734,6 +1735,7 @@ void SurfaceFlinger::rebuildLayerStacks() {
         mVisibleRegionsDirty = false;
         invalidateHwcGeometry();
 
+        bool isLinuxEnable = property_get_bool(PROPERTY_MARUOS_DESKTOP_INTERACTIVE, false);
```

Comment:
Can we rename this variable to `isDesktopInteractive` to make it clearer, since the property name has changed to `PROPERTY_MARUOS_DESKTOP_INTERACTIVE`?